salt/personalised-covers/init.sls

```diff
@@ -64,6 +64,14 @@ personalised-covers-logs:
         - require:
             - file: personalised-covers-cache
 
+personalised-covers-data:
```

Comment:
Ok, by putting files into the formula they can be overridden through Salt (from the central builder-private repository), so this incidentally makes them configurable. This `/formats` should be .gitignored in the project